### ISSUE TITLE
Loeb Moralia - draft MODS & RDF

### ISSUE THREAD

**Issue by cwulfman** [PerseusDL/catalog_data#143]
Here's a first version of an "inverted MODS" - an encoding of the Loeb edition of Plutarch's Moralia. I'd love comments/feedback.

NB The MODS record represents a manifestation (actually a manifestation type, since the record applies to all copies of this edition). It was published in 5 physical volumes; these are represented as <relatedItem type='constituent' otherType='physical'> with part information (a more complete encoding would include the names of the editors and other pubinfo).

The books of the Moralia are logical constituents; they are encoded as <relatedItem type='constituent' otherType='manifestation'> (again, actually a manifestation type); each of these manifestation constituents has an id. Various other information, like locations, could be added.

The RDF does the semantic work. It relates each manifestation to an expression of a work by Plutarch.
The statements about the Works are not shown here; they have already been generated separately; they include an <owl:sameAs> statement relating, say urn:cts:greekLit:tlg0007.tlg068 to http://viaf.org/viaf/234313859, a VIAF authority record for the essay that includes lots of variant names, etc.

So a user wanting to read Plutarch's essay on the education of children would search the semantic data to find expressions of that work; such a search would return the expressions and where they are published.

Let me know what you think; I've started with the hard stuff (the records with hosts), and once we've settled on a format, the work should go pretty quickly.

[moralia-loeb.mods.xml.zip](https://github.com/PerseusDL/catalog_data/files/1837921/moralia-loeb.mods.xml.zip)

[moralia-loeb.rdf.xml.zip](https://github.com/PerseusDL/catalog_data/files/1837870/moralia-loeb.rdf.xml.zip)

**Comment by AlisonBabeu** [comment]
Hi @cwulfman, sorry crazy afternoon here, I will take a look at this in the evening in more detail and offer thoughts on it tomorrow in our meeting if that is all right!

**Comment by cwulfman** [comment]
Sounds good.

**Comment by AlisonBabeu** [comment]
Hi @cwulfman, so I have some additional thoughts on the MODS records, and I know we've talked about how these records can expand to include the page numbers, links, and other important edition level information. This first thought is on possibly encoding some of the information within this MODS record another way. Right now, the following is a typical constituent to express a work:

```xml
 <relatedItem type="constituent"
                   otherType="manifestation"
                   displayLabel="It is Impossible to Live Pleasantly in the Manner of Epicurus">
         <part type="book">
            <detail type="book">
               <caption>Non posse suaviter vivi secundum Epicurum (Ὅτι οὐδὲ ἡδέως ζῆν ἔστιν κατ’ Ἐπίκουρον)</caption>
            </detail>
         </part>
         <identifier type="ctsurn">urn:cts:greekLit:tlg0007.tlg139.loeb-F3</identifier>
      </relatedItem>
```
I'm not sure that we need a displayLabel for the constituents because in this case there is an English and a Latin title displayed in the physical book, and the Greek title was something I added to the original MODS record,and this seems to make the English title somehow the preferred title if that makes any sense. Might it be possible simply to encode the titles for the various constituents within the description as below? (Though in this case we might want to leave out the Greek title as it does not actually appear in the book). 
```xml
     <relatedItem type="constituent"
        otherType="manifestation">
        <part type="book">
           <detail type="book">
              <title lang="eng">It is Impossible to Live Pleasantly in the Manner of Epicurus</title>
              <title lang="lat">Non posse suaviter vivi secundum Epicurum</title>           
              <title lang="grc">Ὅτι οὐδὲ ἡδέως ζῆν ἔστιν κατ’ Ἐπίκουρον</title>
           </detail>
        </part>
        <identifier type="ctsurn">urn:cts:greekLit:tlg0007.tlg139.loeb-F3</identifier>
```
And just one other random thought, loeb-F3, is just a generic edition level identifier yes?

**Comment by AlisonBabeu** [comment]
Actually since it is a Loeb edition, in this case there would be an English and a Greek version, but likely no Latin version. One last question for the day (I promise), why the use of `caption` instead of `title` still?

**Comment by cwulfman** [comment]
Yes -- I was misunderstanding the example you gave. And `title` would be better, you're right.

**Comment by AlisonBabeu** [comment]
I think I just would not like to add yet another element to the mix if we didn't have too! :)

**Comment by cwulfman** [comment]
How about this?

[loeb-mods.xml.zip](https://github.com/PerseusDL/catalog_data/files/1842799/loeb-mods.xml.zip)

**Comment by AlisonBabeu** [comment]
I like how this looks!
